example/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+#vim: et ts =4 sts = 4 sw = 4 tw = 0
+cmake_minimum_required(VERSION 3.1)
+
+set(EXAMPLES
+	readFromString
+	readFromStream
+	stringWrite
+	streamWrite
+   )
+
+add_definitions(-D_GLIBCXX_USE_CXX11_ABI)
+set_property(DIRECTORY PROPERTY COMPILE_OPTIONS $(EXTRA_CXX_FLAGS))
```

Comment:
The problem was actually here — those should be curly braces: `${EXTRA_CXX_FLAGS}`. `g++` parsed `$(EXTRA_CXX_FLAGS)` as a filename, and failed to load the file.